### ISSUE TITLE
The number of training sets does not match when dividing the 9-class.

### ISSUE THREAD

**Issue by LwLiu-2012** [xy980523/ShipsEar-An-Unofficial-Train-Test-Split#6]
Hi, thank you for releasing this train-test split, which has been very helpful for my research.

I'm trying to reproduce the experimental setup from Xu et al. (2023) (https://www.sciencedirect.com/science/article/pii/S0029801823013100), which uses your split and reports a total of 471 training + validation segments (400 training + 71 validation) for the 9-class ShipsEar dataset, using 30-second segments with a 15-second overlap.

However, when I apply the same segmentation strategy to the 9-class training folder, I get 485 segments rather than 471. The test set segment count (116) matches the paper exactly, and the segment counts for the 12-class split (as used in Xie et al., JASA 2024, https://doi.org/10.1121/10.0026598 ) are also fully consistent with the reported numbers.

Could you clarify how the segmentation was performed for the 9-class split?  Any details that could help me reproduce the 471-segment result would be greatly appreciated. Here are the details of how I process each data: 

10__10_07_13_marDeOnza_Sale.wav: 154.86s -> 9 clips
11__10_07_13_minhoUno_Entra.wav: 82.02s -> 4 clips
12__10_07_13_minhoUno_Sale.wav: 121.41s -> 7 clips
14__10_07_13_piraCies_Espera.wav: 60.00s -> 3 clips
16__10_07_13_mscOpera_InicioSalida.wav: 163.23s -> 9 clips
17__10_07_13_visionSub_Entra.wav: 76.89s -> 4 clips
18__18_07_13_AutoPrideEntra.wav: 682.58s -> 44 clips
19__18_07_13_AutoprideMarchaAtras.wav: 374.47s -> 23 clips
21__18_07_13_lanchaMotora.wav: 43.71s -> 1 clips
22__19_07_13_adventure_maniobra.wav: 53.64s -> 2 clips
23__19_07_13_adventure_parado.wav: 9.95s -> 0 clips
25__19_07_13_adventureOfTheSea_llegando.wav: 140.65s -> 8 clips
26__19_07_13_Lancha.wav: 23.49s -> 0 clips
32__19_07_13_Arroios_llega.wav: 216.40s -> 13 clips
33__19_07_13_lanchaDuda_Sale.wav: 37.42s -> 1 clips
34__19_07_13_MarDeOnza_Llega.wav: 123.70s -> 7 clips
36__19_07_13_MiñoUno_Sale.wav: 101.78s -> 5 clips
37__19_07_13_Velero_Sale.wav: 43.98s -> 1 clips
38__19_07_13_Arroios_Sale_2entran.wav: 106.92s -> 6 clips
39__19_07_13_lanchaMotora_Entra.wav: 73.96s -> 3 clips
40__19_07_13_MarDeCangas_Llega_Interf.wav: 44.18s -> 1 clips
41__19_07_13_MiñoUno_Entra.wav: 52.91s -> 2 clips
43__19_07_13_PirataDeCies_Llega_Interf.wav: 51.52s -> 2 clips
45__19_07_13_yate_Sale.wav: 92.66s -> 5 clips
46__23_07_13_H3_bateero1.wav: 11.37s -> 0 clips
47__23_07_13_H2_bateero2.wav: 314.08s -> 19 clips
49__23_07_13_H1_bateero4.wav: 167.00s -> 10 clips
51__23_07_13_H3_lancha2_interf.wav: 100.07s -> 5 clips
52__23_07_13_H3_lancha3_interf.wav: 124.00s -> 7 clips
53__23_07_13_H3_PirataSalvora_Llega1.wav: 161.00s -> 9 clips
54__23_07_13_H3_PirataSalvora_SALE1.wav: 68.05s -> 3 clips
56__23_07_13_H3_velero1.wav: 49.00s -> 2 clips
58__23_07_13_H2_EimskipReefer_Pesquerito1.wav: 239.00s -> 14 clips
59__23_07_13_H2_MarDeMouro.wav: 250.00s -> 15 clips
60__23_07_13_H3_MarDeCangas_LLEGA.wav: 349.00s -> 22 clips
61__23_07_13_H3_MarDeCangas_SALE-2.wav: 143.00s -> 8 clips
63__23_07_13_H3_MarDeOnza_LLEGA.wav: 196.00s -> 12 clips
64__23_07_13_H3_MarDeOnza_SALE.wav: 133.00s -> 7 clips
66__23_07_13_H3_Pesquerito2_Velero.wav: 155.00s -> 9 clips
67__23_07_13_H3_PirataCiesSale.wav: 256.00s -> 16 clips
68__23_07_13_H3_veleroMarisol.wav: 297.00s -> 18 clips
69__23_07_13_H2_costaVoyager.wav: 400.67s -> 25 clips
6__10_07_13_marDeCangas_Entra.wav: 301.36s -> 19 clips
70__23_07_13_H2_yatePeq.wav: 114.40s -> 6 clips
73__23_07_13_H3_pesqMariCarmen.wav: 139.38s -> 8 clips
74__23_07_13_H3_pesqSaladinoPrimero.wav: 145.96s -> 8 clips
76__23_07_13_H3_pesquero2.wav: 177.51s -> 10 clips
77__23_07_13_H3_Planeadora.wav: 92.04s -> 5 clips
79__23_07_13_H3_zodiac.wav: 99.93s -> 5 clips
7__10_07_13_marDeCangas_Espera.wav: 60.00s -> 3 clips
80__04_10_12_adricristuy.wav: 60.24s -> 3 clips
81__25_09_13_H3_corriente.wav: 120.00s -> 7 clips
82__27_09_13_H3_lluvia.wav: 120.00s -> 7 clips
84__27_09_13_H3_viento.wav: 120.00s -> 7 clips
85__E__1L.wav: 85.09s -> 4 clips
86__E__2M.wav: 99.94s -> 5 clips
88__E__4L.wav: 93.94s -> 5 clips
89__E__5M.wav: 93.94s -> 5 clips
8__10_07_13_marDeOnza_Entra.wav: 93.08s -> 5 clips
90__E__6H.wav: 91.95s -> 5 clips
91__E__7H_N.wav: 34.79s -> 1 clips
93__A__Draga_1.wav: 30.27s -> 1 clips
94__A__Draga_2.wav: 50.34s -> 2 clips
96__A__Draga_4.wav: 62.09s -> 3 clips

Total: 485
